src/pages/waiting-room-to-car/__tests__/waiting-room-to-car.analytics.effects.spec.ts

```diff
@@ -113,9 +120,9 @@ describe('Waiting Room To Car Analytics Effects', () => {
       effects.waitingRoomToCarError$.subscribe((result) => {
         expect(result instanceof AnalyticRecorded).toBe(true);
         expect(analyticsProviderMock.logError)
-        // tslint:disable-next-line:max-line-length
+          // tslint:disable-next-line:max-line-length
```

Comment:
can we get rid of this tslint exception as the next line seems smaller then 120 characters